docs/admin/dns.md

```diff
@@ -12,6 +12,8 @@ As of Kubernetes 1.3, DNS is a built-in service launched automatically using the
 A DNS Pod and Service will be scheduled on the cluster, and the kubelets will be
 configured to tell individual containers to use the DNS Service's IP to resolve DNS names.
 
+## What things get DNS names?
+
 Every Service defined in the cluster (including the DNS server itself) will be
```

Comment:
Try to avoid future tense "will be".

"Every Service defined in the cluster (including the DNS server itself) is assigned a DNS name."